packages/modals/src/elements/TooltipDialog/TooltipDialog.tsx

```diff
@@ -194,6 +203,10 @@ TooltipDialogComponent.propTypes = {
   appendToNode: PropTypes.any,
   referenceElement: PropTypes.any,
   placement: PropTypes.any,
+  // @ts-expect-error Validation error due to incorrect type inference when component is wrapped in forwardRef
```

Comment:
Correct type inference for RFCs.
![Screenshot 2025-05-13 at 7 12 48 AM](https://github.com/user-attachments/assets/5d3c17b2-23e4-4f11-ab75-de96f317a03b)

Incorrect type inference for components wrapped in `forwardRef`.  
![Screenshot 2025-05-13 at 7 13 28 AM](https://github.com/user-attachments/assets/a4a17b93-70be-4bb9-94ec-e576ba2cf00d)

🤔

Reply:
Probably why George went with `fallbackPlacements: PropTypes.arrayOf(PropTypes.any)` on `Menu`.